service/src/main/kotlin/xyz/lebkuchenfm/external/storage/mongo/repositories/LlmPersonalityPromptsMongoRepository.kt

```diff
@@ -33,14 +33,14 @@ class LlmPersonalityPromptsMongoRepository(database: MongoDatabase) : LlmPersona
 @Serializable
 private data class LlmPersonalityPromptEntity(
     val text: String,
-    val type: String,
+    val name: String,
```

Comment:
In previous PR I have mistyped this field name. It's correct in the domain model, but wrong here.